modules/programs/git.nix

```diff
@@ -61,7 +61,8 @@ let
   signModule = types.submodule {
     options = {
       key = mkOption {
-        type = types.str;
+        type = types.nullOr types.str;
+        default = null;
```

Comment:
I would remove the default value so users explicitly set it to null.

Reply:
Now removed. The related test is fixed accordingly to this change.